DXRCore/DeusEx/Classes/DXRBase.uc

```diff
@@ -93,7 +95,6 @@ simulated event PreTravel()
     if(dxr != None) {
         _PreTravel();
     }
-    dxr = None;
```

Comment:
maybe we should rename this PreTravel function so it doesn't get called for the modules directly, only from the DXRando actor?